docs/src/languages/php/xdebug.md

```diff
@@ -4,76 +4,109 @@ weight: 6
+You need to have:
 
-Enable Xdebug by adding the following configuration to the application's `.platform.app.yaml` file:
+- PHP 7.2+ on your project
+- The Platform.sh [CLI](../../development/cli/_index.md)
+- Have a Xdebug-compatible IDE installed on your machine.
+    For setup instructions, consult your IDE's Xdebug documentation.
+    Optionally, you can use [PHPStorm](https://www.jetbrains.com/phpstorm/), for which detailed instructions are provided.
 
-```yaml
+## 1. Set up Xdebug
+
+Xdebug runs as a second PHP-FPM process used only for debugging requests, leaving the normal process unaffected.
+
+To enable Xdebug, add the following to your [app configuration](../../create-apps/app-reference.md):
+
+```yaml {location=".platform.app.yaml"}
 runtime:
     xdebug:
         idekey: PHPSTORM
 ```
 
 The `idekey` value can be any arbitrary alphanumeric string.
 
-When that key is defined, Platform.sh will start a second PHP-FPM process on the container that is identically configured but also has Xdebug enabled.  Only incoming requests that have an Xdebug cookie or query parameter set will be forwarded to the debug PHP-FPM process.  All other requests will be directed to the normal PHP-FPM process and thus have no performance impact.
+When that key is defined, Platform.sh starts a second PHP-FPM process on the container that's identically configured but also has Xdebug enabled.
+Only incoming requests that have an Xdebug cookie or query parameter set are forwarded to the debug PHP-FPM process.
+All other requests are directed to the normal PHP-FPM process and thus have no performance impact.
 
-Xdebug has numerous other configuration options available.  They are all set as `php.ini` values, and can be configured the same way as any other [`php.ini` setting](/languages/php/ini.md).  Consult the [Xdebug documentation](https://xdebug.org/docs/) for a full list of available options, although in most cases the default configuration is sufficient.
+If you have enabled the [router cache](../../define-routes/cache.md),
+you need to explicitly add the Xdebug cookie (`XDEBUG_SESSION`) to the cookie allowlist.
+Depending on the cookies already listed, the result should look similar to the following:
 
-If you have the [router cache](../../define-routes/cache.md) enabled, you will also need to explicitly add the Xdebug cookie (`XDEBUG_SESSION`) to the cookie whitelist.  Depending on the cookies you already have listed there the result should look similar to this:
-
-```yaml
+```yaml {location=".platform/routes.yaml"}
 "https://{default}/":
     # ...
     cache:
         enabled: true
         cookies: ['/^SS?ESS/', 'XDEBUG_SESSION']
 ```
 
-## Using Xdebug
+Xdebug has numerous configuration options available.
+They can be set the same way as any other [PHP setting](./_index.md#php-settings).
+For a full list of available options, consult the [Xdebug documentation](https://xdebug.org/docs/).
+
+## 2. Use Xdebug
 
 ### Open a tunnel
```

Comment:
:lady_beetle: The substeps for step 3 have numbers while these don't. It should be consistent. I'd go either unnumbered or letters, as in: https://developers.google.com/style/procedures#sublists